scrapers/AlgoliaAPI/AlgoliaAPI.py

```diff
@@ -191,9 +191,11 @@ def to_scraped_performer(performer_from_api: dict[str, Any], site: str) -> Scrap
     if alternate_names := dig(performer_from_api, "attributes", "alternate_names"):
         performer["aliases"] = alternate_names.strip()
     if height := dig(performer_from_api, "attributes", "height"):
-        performer["height"] = height.strip()
+        performer["height"] = feet_to_cm(height.strip())
     if weight := dig(performer_from_api, "attributes", "weight"):
-        performer["weight"] = weight.strip()
+        performer["weight"] = lb_to_kg(weight.strip())
```

Comment:
In the example https://www.maskurbate.com/en/pornstar/view/Brad/33779, the API has the weight attribute as:

```
"weight": "0.00",
```

This is invalid in the populated field in stash web UI, so you could add a condition for non-zero like:

```python
kg = lb_to_kg(weight.strip())
if kg > 0:
    performer["weight"] = kg
```

and probably do the same approach for `height` and `penis_length`